avocado/plugins/run.py

```diff
@@ -85,6 +85,10 @@ def configure(self, parser):
                             default='off', help='Keep job temporary files '
                             '(useful for avocado debugging). Defaults to off.')
 
+        parser.add_argument('--force-execution', choices=('on', 'off'),
```

Comment:
IMO the description is accurate, but the option name is too generic.  We may want to force execution on other different conditions later.  I'd go with something like `--ignore-resolution-errors`.